src/python/grpcio/grpc/__init__.py

```diff
@@ -1447,6 +1447,19 @@ def stop(self, grace):
         """
         raise NotImplementedError()
 
+    def wait_for_termination(self):
+        """Block current thread until the server stops.
+
+        This is an EXPERIMENTAL API.
+
+        The wait will not consume computational resources during blocking, and it
+        will block indefinitely. There are two ways to unblock:
+
+        1) Calling `stop` on the server in another thread;
+        2) The `__del__` of the server object is invoked.
```

Comment:
`__del__` should appear in the docs. It's an implementation detail and not part of the guaranteed public interface of the object. We can choose to kill `__del__` should we want to.

Reply:
Agreed. Removed.